src/args.jl

```diff
@@ -335,6 +335,12 @@ const _all_defaults = KW[
     _axis_defaults_byletter
 ]
 
+# to be able to reset font sizes to initial values
+const _initial_fontsizes = Dict(:titlefont  => deepcopy(_subplot_defaults[:titlefont]),
```

Comment:
are you sure about that? just doing `Dict(:titlefont  =>_subplot_defaults[:titlefont])` refers to the original type, hence gets changed along with `default(k)`. I tried `copy` but there is no method for the `fonts` type. maybe i dont understand.